docs/eventing/samples/sequence/sequence-terminal/README.md

```diff
@@ -1,6 +1,7 @@
 We are going to create the following logical configuration. We create a
-CronJobSource, feeding events to a [`Sequence`](../../../flows/sequence.md). Sequence
-can then do either external work, or out of band create additional events.
+CronJobSource, feeding events to a [`Sequence`](../../../flows/sequence.md).
```

Comment:
CronJobSource -> Ping source?
Maybe lurking in other places too? Perhaps grep for it to get these updated at the same time?

Reply:
good idea!